engine/src/main/java/org/terasology/rendering/logic/FloatingTextComponent.java

```diff
@@ -25,4 +25,5 @@
     public Color textColor = Color.WHITE;
     public Color textShadowColor = Color.BLACK;
     public float scale = 1f;
+    public boolean isOccluded;
```

Comment:
I'm not sure, but I feel like `isOccluded` is not the best name, because it feels like it is referring to the current state of the text. 'shouldBeOccluded' sounds better, but I'm not sold on the "occluded" part either.
@emanuele3d your thoughts?

Reply:
I was under the same impression, but the Google Code-in task required me to name the field `isOccluded`. A better option IMO would be to use a name like `isDepthTested` which directly relates to the field's function.

Reply:
I realize that the task mentioned `isOccluded` specifically.. but not having written that task myself and after having reviewed this, I'm fairly certain I'm in favour of renaming this.

Reply:
There are two approaches to this. 
1. rename it after what it does from a technical standpoint (depth testing-related)
2. rename it after what it does from a visual standpoint (acts as an overlay or is in-scene)
I'm in favor of option 2 simply because it's quite clear from the code what it does technically. 

So, if it was up to me I'd probably go for something like isOverlay, or isRenderedAsOverlay, effectively flipping the meaning of the variable. That been said, I'm not overly fussed about this.

Reply:
`isOverlay` is a nice choice, much better than `isOccluded`.